__tests__/feature/readNews.test.js

```diff
@@ -0,0 +1,15 @@
+// Copyright 2020 Stanford University see LICENSE for license
+
+import { renderApp } from 'testUtils'
+import { screen } from '@testing-library/react'
+
+describe('reading the latest Sinopia news items', () => {
+  it('displays a div containing the latest news, version of Sinopia, link to github site, and a static text list', async () => {
+    renderApp()
+    await screen.findByText('Latest news')
+    await screen.findByText(/Sinopia Version \d\.\d\.\d highlights/)
+    screen.getByRole('link', { name: 'Sinopia help site' })
+    const list = screen.getAllByRole('listitem')
+    expect(list.length).not.toBeLessThan(4)
```

Comment:
I think the exact number of items depends on @michelleif for each major or minor version updates. I think we will always have at least one item.